code/API_definitions/Edge-Application-Management.yaml

```diff
@@ -925,6 +931,38 @@ components:
         - unknown
       default: unknown
 
+    EdgeCloudZoneFlavors:
```

Comment:
There is another parameter "Flavor" below in yaml. How does that correlates with EdgeCloudZoneFlavors?